### ISSUE TITLE
Web UI doesn't render packaged dag code properly

### ISSUE THREAD

**Issue by dalex100** [apache/airflow#10554]
**Apache Airflow version**: 1.10.10

**Kubernetes version (if you are using kubernetes)** (use `kubectl version`): Client : 1.15.5 / Server : v1.16.13-eks-2ba888

**Environment**:

- **Cloud provider or hardware configuration**: AWS EKS
- **OS** (e.g. from /etc/os-release):
- **Kernel** (e.g. `uname -a`):
- **Install tools**:
- **Others**:

**What happened**:
Using **serialized_dags=true** and s**tore_dag_code=true** with packaged dags, it is not possible to view the dag code in the web UI. (data is blurred in case there's sensitive information). Packaged dags render fine when dags aren't serialized.

![image](https://user-images.githubusercontent.com/15349267/91212126-cf66c080-e6dd-11ea-915f-9eb8a968d77b.png)

**What you expected to happen**:
The web UI should display the dag code.

**How to reproduce it**:
Use the following settings : 

```
AIRFLOW__CORE__STORE_SERIALIZED_DAGS: "true"
AIRFLOW__CORE__STORE_DAG_CODE: "true"
```

and upload a packaged dag into Airflow.

**Anything else we need to know**:

**Comment by potiuk** [comment]
@kaxil @anikarni -> that might be indeed an interesting problem with DAG serialization

**Comment by kaxil** [comment]
I will definitely take a look, thanks for creating the issue @dalex100

**Comment by potiuk** [comment]
I guess we have not addressed it in 2.1.1 - moving it to 2.2

**Comment by kaxil** [comment]
I was not able to replicate this on Main branch, looks like it was fixed by https://github.com/apache/airflow/pull/13984 which was  released in 2.1.0
